metrics/web/test/features/auth/domain/usercases/sign_out_usecase_test.dart

```diff
@@ -11,5 +13,18 @@ void main() {
         MatcherUtil.throwsAssertionError,
       );
     });
+
+    test('signs out user on call', () async {
+      final repository = UserRepositoryStub(seedUser: User(id: 'id'));
+      final signOutUseCase = SignOutUseCase(repository);
+      final currentUser = await repository.currentUserStream().first;
+
+      expect(currentUser, isNotNull);
+
+      await signOutUseCase();
+      final newUser = await repository.currentUserStream().first;
+
+      expect(newUser, isNull);
+    });
```

Comment:
As in sign test we should check that method is called and that's it, if any exceptions could be thrown - we should test that as well using stub repository.